OrbitGrpcProtos/capture.proto

```diff
@@ -78,6 +78,22 @@ message CallstackSample {
   uint64 timestamp_ns = 5;
 }
 
+message InternedTracepointInfo {
+  uint64 key = 1;
+  TracepointInfo intern = 2;
+}
+
+message TracepointSample {
```

Comment:
What about calling this `TracepointEvent`? We tend to use "sample" when we refer to time-based callstack samples.